var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -105,4 +110,13 @@ def cmake_args(self):
         else:
             options.append('-DGMX_USE_RDTSCP:BOOL=ON')
 
+        if '+mdrun_only' in self.spec:
+            options.append('-DGMX_BUILD_MDRUN_ONLY:BOOL=ON')
+
+        if '~openmp' in self.spec:
+            options.append('-DGMX_OPENMP:BOOL=OFF')
+
+        if '+double_precision' in self.spec:
+            options.append('-DGMX_RELAXED_DOUBLE_PRECISION:BOOL=ON')
```

Comment:
I always like to add an `else` clause that explicitly disables options. That way, if the default changes or CMake tries to be fancy and sets the default based on what your compiler is capable of or what dependencies it finds on your system, the build is still reproducible.

Reply:
Thank you for your reply!

I added the `else` clauses that explicitly  disable option and the `else` clause that explicitly enables option too.